src/System.IO.Ports/src/System/IO/Ports/SerialPort.Win32.cs

```diff
@@ -11,6 +11,8 @@ namespace System.IO.Ports
 {
     public partial class SerialPort : Component
     {
+        const string GUID_DEVINTERFACE_COMPORT = "86e0d1e0-8089-11d0-9ce4-08003e301f73"; // SerialPort GUID Class ID
```

Comment:
nit: explicit `private`
nit2: comment above the line
nit3: if the constant is not defined elsewhere with this exact casing please use UsualCasing